types/index.d.ts

```diff
@@ -804,6 +804,12 @@ declare namespace RamdaAdjunct {
          */
         allP<T>(iterable: Iterable<T>): Promise<T[]>;
 
+        /**
+         * Returns a Promise that is resolved with an array of reasons when all of the provided Promises reject, or rejected when any Promise is resolved.
+         * This pattern is like allP, but fulfillments and rejections are transposed - rejections become the fulfillment values and vice versa.
+         */
+        noneP<T>(iterable: Iterable<T | Promise<T>>): Promise<T[]>;
```

Comment:
@char0n - actually do you think it would make more sense if the return type was `Promise<any[]>` instead of `Promise<T[]>`? I am thinking that the rejection reasons could be anything (i.e. `any`) and the promise will resolve with the rejection reasons since the fulfillments and rejections are transposed 🤔

Reply:
Can we have definitions ? One for rejection one for resolution case. Notice that `any` is not an array here. This will be verified in typings tests I guess.
```typescript
noneP<T>(iterable: Iterable<T | Promise<T>>): Promise<T[]>;
noneP<T>(iterable: Iterable<T | Promise<T>>): Promise<any>;
```